src/test/ui/lang-items/generics/wrong-number-fn.rs

```diff
@@ -0,0 +1,14 @@
+#![feature(lang_items,no_core)]
```

Comment:
Can all those tests be put in a single file?

Reply:
I had to make it two, because I have two tests that should pass for a couple different lang items, and one for the ones that fail.  But they have been merged into two files in the `lang-items` directory root.